lib/attributes_json.rb

```diff
@@ -15,7 +15,10 @@ module FormatParser::AttributesJSON
 
   # Implements a sane default `as_json` for an object
   # that accessors defined
-  def as_json(root: false)
+  #
+  # stringify_keys: transforms all the hash keys to a string when is true. The
+  # default value is false for backward compatibility
+  def as_json(root: false, stringify_keys: false)
```

Comment:
Are we sure we support all the options here which ActiveSupport offers? We probably need to add a capture for extra keyword arguments with a `**` for the case that Rails adds another option to `as_json`

Reply:
Yeah, we don't support all the options, but should we support it?

I didn't find any issue about it and it has been like this for a long time!

Reply:
> but should we support it?
Good question - not that we should support them all, but I think we should not crash if they are supplied to us

Reply:
ok, I added the `**`!